tracks/rust/exercises/minesweeper/mentoring.md

```diff
@@ -0,0 +1,61 @@
+### Concepts
+
+- Iterators
+- Iterator Methods
+- &str
+- char
+
+### Reasonable Solutions
+
+A reasonable solution should:
+
+- Use iterator methods instead of slice indexing whenever possible
+- Avoid unnecessary heap allocation
+- Handle edge cases (first and last line / column) gracefully
+- ***optional***: Boost readability by e.g. defining mine character ('*') as a constant
```

Comment:
```suggestion
- ***optional***: Boost readability by e.g. defining mine character (`*`) as a constant
```

Reply:
I meant to use the syntax of character literals in Rust, but I guess `*` or `'*'` is better rendered on the webpage.

Reply:
Yep! So any _syntax_, I would suggest put in a code block/inline-block. `'*'` would be perfectly fine!